packages/react/src/xcss-prop/index.ts

```diff
@@ -30,18 +30,29 @@ type XCSSPseudo<
     : never;
 };
 
+type XCSSMediaQuery<
+  TAllowedProperties extends keyof StrictCSSProperties,
+  TAllowedPseudos extends CSSPseudos,
+  TAllowedMediaQueries extends string,
+  TSchema
+> = {
+  [Q in `@media ${TAllowedMediaQueries}`]?:
+    | XCSSValue<TAllowedProperties, TSchema, ''>
+    | XCSSPseudo<TAllowedProperties, TAllowedPseudos, never, TSchema>;
+};
+
 /**
  * These APIs we don't want to allow to be passed through the `xcss` prop but we also
  * must declare them so the (lack-of a) excess property check doesn't bite us and allow
  * unexpected values through.
  */
-type BlockedRules = {
+type BlockedRules<TMode extends 'loose' | 'strict'> = {
+  // To ensure styles that aren't allowed through XCSS prop strict APIs we block any
+  // loose media queries from being passed through as we can't ensure they're correct.
+  '@media [loose]'?: TMode extends 'loose' ? any : never;
```

Comment:
Remember this is for the input to xcss prop, so for loose we accept anything.

Reply:
You could use something like `@media not all` if you wanted it to be a bit more _semantic_ 🤣

Reply:
I didn't want it to be semantic haha, didn't want it to be valid!